package.json

```diff
@@ -16,7 +16,7 @@
     "format": "prettier --write .",
     "format:check": "prettier --check .",
     "build:lib:prod": "ng build ngx-datatable-lib --configuration production",
-    "copy-files": "cp README.md dist/ngx-datatable/README.md && cp LICENSE dist/ngx-datatable/LICENSE && cpx \"src/assets/**/*\" \"dist/ngx-datatable/assets\" && cpx \"projects/ngx-datatable/src/lib/themes/**/*\" \"dist/ngx-datatable/themes\"",
+    "copy-files": "cpy \"{LICENSE,README.md}\" dist/ngx-datatable/ && cpy \"src/assets/**/*\" \"dist/ngx-datatable/assets\"",
```

Comment:
While this looks ok to me on the first sight, I did a quick comparison of the `dist/` folder with and without this change after runnging `rm -rf dist/ && npm run build && npm run package` and we seem to miss a few files that are somewhat required:

<img width="1111" alt="image" src="https://github.com/user-attachments/assets/f513ba0b-aa61-40d8-922e-6dce31102e06" />

Reply:
Can you please check again? For me those files are there:

![image](https://github.com/user-attachments/assets/2bbd0910-cc18-4ef2-b2e8-66aad4660107)
Same also for this dist folder when downloading from the pipeline here